kinto/plugins/accounts/views.py

```diff
@@ -4,6 +4,8 @@
 from pyramid.security import Authenticated, Everyone
 from pyramid.settings import aslist
 
+from ..hawk.hawkauth import HawkAuth
```

Comment:
use absolute imports `from kinto.plugins.hawk`

Reply:
thanks for the tip!  Is this better style or is there another reason to avoid relative imports here?

Reply:
I think it's better for readability and style, but in this case, it's mainly for consistency with the rest of the codebase. I think we only have relative imports in tests :)